src/stories/Quiz/utils/StoryPreview.tsx

```diff
@@ -3,20 +3,19 @@ import StoryVariations from './StoryVariations';
 
 export interface IStoryPreviewProps {
   Component: (props: any) => JSX.Element;
-  variationsArgsList: any[];
 }
 
 export default function StoryPreview(props: IStoryPreviewProps) {
-  const { Component, variationsArgsList } = props;
+  const { Component } = props;
```

Comment:
Removed `variationsArgsList` Since the components are dependent on the Context not props